### PR TITLE
[W5.11][F14-A2] Marcus Chen

### PR DESCRIPTION
**Pull request by Marcus-cxc**
Enhancement allows find command to be insensitive

New function added to convert a list of string into lowercased string version
Keywords and name are converted before comparing hence insensitive
Case of the name stored is not modified

### CODE REVIEW

**Comment by wynonaK** [comment]
Interesting idea. Probably a suggestion would be to store it in a format of which you wont have a to modify both sides, but only one side.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (cjianhui) [inline src/seedu/addressbook/commands/FindCommand.java:3]: While it appears more efficient to group all the removed imports under `java.util.*`, imported classes should always be listed explicitly as mentioned in our [coding standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html).

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -49,12 +46,24 @@ public CommandResult execute() {
...
         return matchedPersons;
     }
-
+    /**
+     * Converts an array of string into its lower cased version
+     *
+     * @param toConvert source ArrayList<String)
+     * @return All string in array converted to lower cased
+     */
+    private static List<String> convertStringListToLowerCase (List<String> toConvert) {
```
Comment (cjianhui) [inline src/seedu/addressbook/commands/FindCommand.java:62]: Just a minor comment but another alternative method name you could have used was convertStringsToLowerCase.

**Review by cjianhui — COMMENTED** [review]
@Marcus-cxc I have added some comments. Please close the PR after reading them.

Good job on updating the documentation and tests. Your Travis probably failed because of the missing JUnit test but not to worry as we will be going through it soon.

**Pull request closed without merging**